packages/instantsearch-ui-components/src/components/chat/ChatMessage.tsx

```diff
@@ -297,27 +246,42 @@ export function createChatMessageComponent({
         <div className={cx(cssClasses.container)}>
           {hasLeading && (
             <div className={cx(cssClasses.leading)}>
-              {LeadingComponent ? (
-                <LeadingComponent />
-              ) : (
-                AvatarComponent && <AvatarComponent />
-              )}
+              {LeadingComponent && <LeadingComponent />}
             </div>
           )}
 
           <div className={cx(cssClasses.content)}>
             <div className={cx(cssClasses.message)}>
               {message.role === 'assistant'
-                ? message.parts.map(renderMessagePart)
-                : message.parts.map(renderMessagePart)}
+                ? message.parts.map(renderAssistantPart)
+                : message.parts.map(renderAssistantPart)}
```

Comment:
looks like this may have been a wrong rebase. I think it should be `renderMessagePart` or `renderPart`

Reply:
By the way, as a performance optimization, should we split it into two functions and render the user message as a string?